sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -570,7 +568,10 @@ int gnrc_netif_ipv6_addr_add_internal(gnrc_netif_t *netif,
         DEBUG("nib: Can't join solicited-nodes of %s on interface %u\n",
               ipv6_addr_to_str(addr_str, addr, sizeof(addr_str)),
               netif->pid);
+        return res;
     }
+    netif->ipv6.addrs_flags[idx] = flags;
+    memcpy(&netif->ipv6.addrs[idx], addr, sizeof(netif->ipv6.addrs[idx]));
```

Comment:
I think these 2 lines should be below the `#endif /* GNRC_IPV6_NIB_CONF_ARSM */`

Reply:
Oh yeah!

Reply:
(that's the problem with last-minute optimizations you make shortly before pushing, but *after* testing xD)

Reply:
No wait, even if they are moved belof that `#endif`, they are still inside the `  #ifdef MODULE_GNRC_IPV6_NIB` (I think).

Reply:
Yes. Fixed that as well ;-).